sympy/vector/coordsysrect.py

```diff
@@ -10,7 +10,16 @@
                                     SpaceOrienter, QuaternionOrienter)
 
 
-class CoordSysCartesian(Basic):
+def CoordSysCartesian(*args, **kwargs):
+    SymPyDeprecationWarning(
```

Comment:
Also please add it to https://github.com/sympy/sympy/wiki/Release-Notes-for-1.1 (and add any other changes you know of for this release to that document).